src/base/rss/rss_autodownloader.cpp

```diff
@@ -245,6 +245,7 @@ void AutoDownloader::processJob(const QSharedPointer<ProcessingJob> &job)
         }
 
         rule.setLastMatch(job->articleDate);
+        rule.matched(job->articleTitle);
```

Comment:
Another thing I don't like are "previously matched episode" word combination in interface method names (as well as currently used "last match"). One gets the impression that we fit in class internals. It would be nice to use a more appropriate concept. I'm not forcing you to do it here - it's just my thoughts. 
Maybe something like "exclude episode name"...